### ISSUE TITLE
seek help

### ISSUE THREAD

**Issue by 2770849306** [ankit5902/hp-probook-440-g2#1]
Thank you for sharing，I would like to ask, can this OC boot MAC os10.15.6? Why can my laptop boot MAC os10.13.6 at most? What should be set for BIOS

**Comment by ankit5902** [comment]
Yes, it can boot macOS 10.15.6.  Could you tell me what problem are you facing?
**As for BIOS you need to set Load your default factory setting (this is to clean any previous setting you have and start with clean slate)**
- Disable CPU VT-d (VT-d enable access to hardware from inside virtual machine)
- Disable Secure Boot
- Enable XHCI Handoff
- Change OS Type to Other (If your BIOS has this option)
- Enable iGPU if you want to use it/ Disable if you’re using external card
- Disable Fast Boot
- Disable CSM Support (If your BIOS has this option)
- Change SATA Mode selection to AHCI

**Also you need to provide your Platform Info in config.plist (since it is edited out in my version)**

**Comment by 2770849306** [comment]
However, when I install MacOS10.15.6, the code is stuck at https://panic.apple.com to report this panic.
